src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/UnitTestingIdleProcessor.cs

```diff
@@ -117,6 +118,9 @@ protected async Task<bool> WaitForIdleAsync(IExpeditableDelaySource expeditableD
 
     private async Task ProcessAsync()
     {
+        // Ensurew we do this processing on the background.
+        await Task.Yield().ConfigureAwait(false);
```

Comment:
`await TaskScheduler.Default;` not available here as we're at teh Features layer and don't have access to that VS extension.